src/renderer/services/neo.js

```diff
@@ -828,6 +836,12 @@ export default {
       });
     }
 
+    const gasClaim = {
+      step: 0,
+    };
+    store.commit('setGasClaim', gasClaim);
+    store.commit('setShowGasClaimModal', true);
```

Comment:
Does committing this mean it will persist across shutting down the app and coming back?

Reply:
Tested, and it does not persist across shutting down and coming back; which is fine since it has the warning.